tests/test-core/utils/js/ConditionalTestUtils.js

```diff
@@ -83,43 +83,30 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
         });
     };
 
-    // Adds context awarenes of the browser's reported platform (operating
-    // system) for aiding in executing platform-specific tests, or skipping
-    // those known to fail on a particular platform due to lack of browser
-    // support for features
-
     // Checks string-based contextAwareness check values to see if they
     // contain the searchValue string
     fluid.test.conditionalTestUtils.contextValueContains = function (searchValue, checkValue) {
         var value = fluid.contextAware.getCheckValue(fluid.rootComponent, checkValue);
         return value.indexOf(searchValue) >= 0;
     };
 
-    fluid.test.conditionalTestUtils.getPlatformName = function () {
-        return navigator.platform ? navigator.platform : undefined;
-    };
-
-    // Functions for platform reporting for makeChecks
-    fluid.test.conditionalTestUtils.isLinux = function () {
-        return fluid.test.conditionalTestUtils.contextValueContains("Linux", "{fluid.platform}");
+    // Functions for browser platform reporting for makeChecks
+    fluid.test.conditionalTestUtils.isBrowserLinux = function () {
+        return fluid.test.conditionalTestUtils.contextValueContains("Linux", "{fluid.browserPlatformName}");
     };
 
-    fluid.test.conditionalTestUtils.isMac = function () {
-        return fluid.test.conditionalTestUtils.contextValueContains("Mac", "{fluid.platform}");
+    fluid.test.conditionalTestUtils.isBrowserMac = function () {
+        return fluid.test.conditionalTestUtils.contextValueContains("Mac", "{fluid.browserPlatformName}");
     };
 
-    fluid.test.conditionalTestUtils.isWindows = function () {
-        return fluid.test.conditionalTestUtils.contextValueContains("Windows", "{fluid.platform}");
+    fluid.test.conditionalTestUtils.isBrowserWindows = function () {
+        return fluid.test.conditionalTestUtils.contextValueContains("Windows", "{fluid.browserPlatformName}");
     };
 
     fluid.contextAware.makeChecks({
-        "fluid.platform": "fluid.test.conditionalTestUtils.getPlatformName"
-    });
-
-    fluid.contextAware.makeChecks({
-        "fluid.platform.isLinux": "fluid.test.conditionalTestUtils.isLinux",
-        "fluid.platform.isMac": "fluid.test.conditionalTestUtils.isMac",
-        "fluid.platform.isWindows": "fluid.test.conditionalTestUtils.isWindows"
+        "fluid.platform.isBrowserLinux": "fluid.test.conditionalTestUtils.isBrowserLinux",
+        "fluid.platform.isBrowserMac": "fluid.test.conditionalTestUtils.isBrowserMac",
+        "fluid.platform.isBrowserWindows": "fluid.test.conditionalTestUtils.isBrowserWindows"
```

Comment:
Similar to above, can we use `fluid.browser` namespace instead. e.g. `fluid.browser.platform.isLinux`